uber/config.py

```diff
@@ -196,8 +196,11 @@ def PAGE(self):
 
     @property
     def CURRENT_ADMIN(self):
-        with sa.Session() as session:
-            return session.admin_attendee().to_dict()
+        try:
+            with sa.Session() as session:
+                return session.admin_attendee().to_dict()
+        except:
+            return {}
```

Comment:
Simple enough fix, though in the long run we might want to instead fix ``magfest.js`` to be more fault tolerant.